deployment/streaming/docker-compose.deploy.yml.tpl

```diff
@@ -0,0 +1,44 @@
+version: '3.0'
+services:
+  augdiff-stream:
+    image: ${ECR_REPO}:latest
+    command: >
+      /spark/bin/spark-submit --driver-memory 4096m --class osmesa.analytics.oneoffs.AugmentedDiffStreamProcessor /opt/osmesa-analytics.jar
+      --augmented-diff-source ${AUGDIFF_SOURCE}
+      --start-sequence ${AUGDIFF_START}
+      --database-uri ${DB_URI}
+      --database-user ${DB_USER}
+      --database-pass ${DB_PASS}
+    deploy:
+      restart_policy:
+        condition: on-failure
+        delay: 1s
+        max_attempts: 10
+        window: 120s
+    logging:
+      driver: awslogs
+      options:
+        awslogs-group: ${AWS_LOG_GROUP}
+        awslogs-region: ${AWS_REGION}
+        awslogs-stream-prefix: augdiff
+  changeset-stream:
+    image: ${ECR_REPO}:latest
+    command: >
+      /spark/bin/spark-submit --driver-memory 4096m --class osmesa.analytics.oneoffs.ChangesetStreamProcessor /opt/osmesa-analytics.jar
+      --changeset-source ${CHANGESET_SOURCE}
+      --start-sequence ${CHANGESET_START}
+      --database-uri ${DB_URI}
+      --database-user ${DB_USER}
+      --database-pass ${DB_PASS}
```

Comment:
User / password are no longer necessary as separate args, right? (Same for above.)